cran-comments.md

```diff
@@ -1,20 +1,13 @@
 ## Test environments
 
-release 4.0.4
+release 4.0.3
 
 * OSX (local) - release
 * OSX (actions) - release
-* Ubuntu (actions) - 3.5, 3.6, oldrel, release and devel
+* Ubuntu (actions) - 3.4 to 3.6, oldrel, release and devel
 * Windows (actions) - release
 * Windows (winbuilder) - devel
 
 ## R CMD check results
 
 0 errors | 0 warnings | 0 notes
-
```

Comment:
ssdtools has :	GenHMM1d as a reverse dependency - we need to check this and indicate in the cran-comments